src/FixedDataTableCellDefault.js

```diff
@@ -87,26 +87,14 @@ class FixedDataTableCellDefault extends React.Component {
       <div
         {...props}
         className={joinClasses(
-          cx('fixedDataTableCellLayout/wrap1'),
-          cx('public/fixedDataTableCell/wrap1'),
+          cx('fixedDataTableCellLayout/wrap'),
```

Comment:
We condense all the 3 wrapper levels into the top level div.
Also I'm pulling up the `cellContent` class name.